src/Components/Artwork/Metadata.tsx

```diff
@@ -25,6 +25,7 @@ export interface MetadataProps
   showSaveButton?: boolean
   to?: string | null
   renderSaveButton?: (artworkId: string) => React.ReactNode
+  saveOnlyToDefaultList?: boolean
```

Comment:
Can avoid all of the passing via https://github.com/artsy/force/blob/c9a511068a3870c78756eb80fc877f93eccc5622/src/Components/ArtworkGrid/ArtworkGridContext.tsx#L21

Reply:
thanks @damassi ! Fixed